javascript/javascript2/week3/warmup.js

```diff
@@ -0,0 +1,100 @@
+//1. Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+console.log("text will display after 4 seconds");
+setTimeout (() => {
+    console.log("4 seconds finished....hello");},2000);
+    
+//2. Create a function that takes 2 parameters: delay and stringToLog. 
+function timeDelay(delay,StringtoLog)
+{
+    setTimeout (() => {
+        console.log(StringtoLog);},delay);      
+}
+timeDelay(2000, "stringToLog after delay seconds from function ");
+
+/*3. Create a button in html. When clicking this button, use the function you created 
+in the previous task to log out the text: Called after 5 seconds 5 seconds after 
+the button is clicked.*/
+
+const JsDelayBtn = document.getElementById('DelayBtn');
+JsDelayBtn.addEventListener('click', timeDelay1(5000, "Text after given time delay "));
+function timeDelay1 (delay,StringtoLog)
+{
+    setTimeout (() => {
+        document.getElementById("root").innerHTML = (StringtoLog);},delay);;  
+}
+
```

Comment:
ypu don't need to create new function timeDelay1 but reuse the function from previous task